sigh/lib/assets/resign.sh

```diff
@@ -201,8 +201,9 @@ then
 fi
 
 # Set the app name
-# The app name is the only file within the Payload directory
-APP_NAME=$(ls "$TEMP_DIR/Payload/")
+# In Payload directory may be another file except .app file, such as StoreKit folder.
+# Search the first .app file within the Payload directory
+APP_NAME=$(ls "$TEMP_DIR/Payload/"|grep ".app$"| head -1)
```

Comment:
@lacostej Thanks.i think your codes work better. But i think there will never happen after unzip the ipa file which can be installed.Could you tell about your case.